### DISCUSSION TITLE
Determine the touch input type, distinguishing between finger and stylus.

### DISCUSSION THREAD

**Discussion by amitsri007** [software-mansion/react-native-gesture-handler#2735] [Q&A] [answered]
How to determine the touch type between finger and stylus and the pressure of the stylus. I want to make a drawing app where users use a stylus to draw lines and fingers to pan, move, and zoom canvas.

**Comment by m-bert** [comment, marked as answer]
Hi @amitsri007! We thought about it a while ago, but your post made us decide that it will be a good idea. It should be available after we merge [this PR](https://github.com/software-mansion/react-native-gesture-handler/pull/2760).

**Reply by amitsri007** [reply]
@m-bert Great, Thanks. I am looking forward to seeing the changes in action.
